modules/packages/manifests/mozilla/generic_worker.pp

```diff
@@ -33,6 +34,14 @@
                     owner  => root,
                     group  => wheel,
             }
+            # install quarantine-worker, Bug 1461913
+            file {
+                '/usr/local/bin/quarantine-worker':
```

Comment:
For generic-worker there is a comment about where binaries should be downloaded from. Can you add a similar comment for quarantine-worker?